src/shorthands.jl

```diff
@@ -104,3 +104,215 @@ zlims!(lims::NTuple{2, Real}) = zlims!(current_scene(), lims)
+"""
+ztickrange(scene) = tickranges(scene)[3]
+
+"""
+    ticks!([scene,]; tickranges=tickranges(scene), ticklabels=ticklabels(scene))
+
+Set the tick labels and ranges along all axes. The respective labels and ranges
+along each axis must be of the same length.
+"""
+function ticks!(scene; tickranges=tickranges(scene), ticklabels=ticklabels(scene))
+    axis = scene[Axis]
+    @assert !isnothing(axis)
+    # Have to set `ranges_labels` first so that any changes in the length of these
+    # is reflected there first.
+    axis.ticks.ranges_labels[] = (tickranges, ticklabels)
+    axis.ticks.ranges[] = tickranges
+    axis.ticks.labels[] = ticklabels
+    return nothing
+end
+function ticks!(; tickranges=tickranges(scene), ticklabels=ticklabels(scene))
+    ticks!(current_scene(), ranges=ranges, labels=labels)
+end
+
+"""
+    xticks!([scene,]; xtickranges=xtickrange(scene), xticklabels=xticklabel(scene))
+
+Set the tick labels and range along the x-axes. See also [`ticks!`](@ref).
+"""
+function xticks!(scene; xtickrange=xtickrange(scene), xticklabels=xticklabels(scene))
+    ticks!(scene, tickranges=(xtickrange, tickranges(scene)[2:end]...), ticklabels=(xticklabels, ticklabels(scene)[2:end]...))
+    return nothing
+end
+function xticks!(; xtickrange=xtickrange(scene), xticklabels=xticklabels(scene))
+    xticks!(current_scene(), xtickranges=xtickrange, xticklabels=xticklabels)
+end
+
+"""
+    yticks!([scene,]; ytickranges=ytickrange(scene), yticklabels=yticklabel(scene))
+
+Set the tick labels and range along all the y-axis. See also [`ticks!`](@ref).
+"""
+function yticks!(scene; ytickrange=ytickrange(scene), yticklabels=yticklabels(scene))
+    r = tickranges(scene)
+    l = ticklabels(scene)
+    if length(r) == 2
+        ticks!(scene, tickranges=(first(r), ytickrange), ticklabels=(first(l), yticklabels))
+    else  # length(r) == 3
+        ticks!(scene, tickranges=(first(r), ytickrange, last(r)), ticklabels =(first(l), yticklabels, last(l)))
+    end
+    return nothing
+end
+function yticks!(; ytickrange=ytickrange(scene), yticklabels=yticklabels(scene))
+    yticks!(current_scene(), ytickrange=ytickrange, yticklabels=yticklabels)
+end
+
+"""
+    zticks!([scene,]; ztickranges=ztickrange(scene), zticklabels=zticklabel(scene))
+
+Set the tick labels and range along all z-axis. See also [`ticks!`](@ref).
+"""
+function zticks!(scene; ztickrange=ztickrange(scene), zticklabels=zticklabels(scene))
+    ticks!(scene, tickranges=(tickranges(scene)[1:2]..., ztickrange), ticklabels=(ticklabels(scene)[1:2]..., zticklabels))
+    return nothing
+end
+function zticks!(; ztickrange=ztickrange(scene), zticklabels=zticklabels(scene))
+    zticks!(current_scene(), ztickrange=ztickrange, zticklabels=zticklabels)
+end
+
+###
+### Ticks rotations
+###
+"""
+    ticksrotations(scene)
+
+Returns the rotation of all tick labels.
+"""
+function ticksrotations(scene)
+    axis = scene[Axis]
+    @assert !isnothing(axis)
```

Comment:
```suggestion
    @assert !isnothing(axis) "The Scene does not have an axis!"
```

Reply:
Would you like me to add this to all of the functions in this file, not jus the methods I'm contributing.

Reply:
May as well...thanks!